main.go

```diff
@@ -306,17 +304,20 @@ func getKeyStore(datastore string, keyName string) *Keystore {
 	return NewKeystore(dirstore, keyName)
 }
 
-func loadTenantToken(datastore string) ([]byte, error) {
+func loadTenantToken(config *menderConfig, datastore string) ([]byte, error) {
+
 	dirstore := NewDirStore(datastore)
-	raw, err := dirstore.ReadAll(defaultTenantTokenFile)
+
+	raw, err := dirstore.ReadAll(config.GetTenantToken())
```

Comment:
* `DeviceKey` - I agree, there is absolutely no need for this to be configurable. Feel free to remove this where it's used and replace with a const.
* `TenantTokenPath` - This will turn into an actual token, like @oleorhagen described, so this is a non-issue.